onnxruntime/core/providers/dml/OperatorAuthorHelper/OperatorHelper.cpp

```diff
@@ -57,14 +57,15 @@ namespace OperatorHelper
     }
 
     template <typename T>
-    void ExpandToAxes(/*inout*/ std::vector<T>& attr, std::vector<int32_t> axes, std::vector<T> expanded)
+    void ExpandToAxes(/*inout*/ std::vector<T>& originalValues, gsl::span<int32_t> axes, std::vector<T> expanded)
```

Comment:
`gsl::span<int32_t> axes` - pass as const view, since they are not modified
`originalValues = expanded;` - better to std::move it to avoid reallocation.
Rest LGTM.